src/App.tsx

```diff
@@ -1,167 +1,43 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { Outlet, NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export const App = () => {
+  const html = document.getElementsByTagName('html')[0];
```

Comment:
Directly manipulating the DOM to add a class to the HTML element is not recommended in React. Consider using a React effect hook, such as `useEffect`, to handle this side effect.